### PR TITLE
updating scripts structure

### PR DESCRIPTION
**Pull request by mohamedamer453**
### Description

Update the structure of all scripts in the grid client.

### Changes

- Changed the scripts to have a function for each functionality and this function get called in the main function according to the structure mentioned in #950.
- Updated the structure of each script to use the model directly to provide autocomplete for the configuration and to show the proper error message if a configuration option is missing.

### Related Issues

#950 

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/grid_client/scripts/blockchain.ts [hunk 1]
```diff
@@ -82,13 +82,16 @@ async function main() {
   await grid3.disconnect();
 }
 
-async function createAccount(account_name: string, blockchain_type: blockchainType, ip?: string) {
+async function createAccount(account_name: string, blockchain_type: blockchainType) {
   try {
-    const account_created = await grid3.blockchain.create({
+    //change the relay to the desired network
+    const relay = "relay.dev.grid.tf";
+    const account: BlockchainCreateModel = {
```
Comment (maayarosama) [inline packages/grid_client/scripts/blockchain.ts:89]: Please make sure to initialize the model in the main function and pass it as a parameter to the other function as you did in all the other scripts.

File: packages/grid_client/scripts/algorand.ts [hunk 2]
```diff
@@ -31,40 +42,44 @@ async function main() {
 
 async function createAccount(account_name: string) {
   try {
-    const account_created = await grid3.algorand.create({
+    const account_created: AlgorandAccountCreateModel = {
```
Comment (maayarosama) [inline packages/grid_client/scripts/algorand.ts:45]: Please make sure to initialize the model in the main function and pass it as a parameter to the other function as you did in all the other scripts.

**Review by maayarosama — CHANGES_REQUESTED** [review]

File: packages/grid_client/scripts/algorand.ts (hunk 2 above)
Comment (mohamedamer453) [reply packages/grid_client/scripts/algorand.ts]: should be fixed in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1409/commits/cba9251ff5af6781eab04857ecdc1079490b9654

File: packages/grid_client/scripts/blockchain.ts (hunk 1 above)
Comment (mohamedamer453) [reply packages/grid_client/scripts/blockchain.ts]: should be fixed in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1409/commits/8fc5cb9cad2588dbc9c9a3bb91da3c90db688e6b

**Review by maayarosama — APPROVED** [review]

File: packages/grid_client/scripts/algorand.ts [hunk 3]
```diff
@@ -1,102 +1,142 @@
...
-  await deleteAccount("myaccount");
+  await transferAccount(transfer);
+  await deleteAccount(deleteAccount1);
+  await deleteAccount(deleteAccount2);
 
   grid3.disconnect();
 }
 
-async function createAccount(account_name: string) {
+async function createAccount(account) {
```
Comment (AlaaElattar) [inline packages/grid_client/scripts/algorand.ts:89]: why there're no comments here in all functions like the rest of scripts?

**Review by AlaaElattar — APPROVED** [review]

File: packages/grid_client/scripts/algorand.ts (hunk 3 above)
Comment (mohamedamer453) [reply packages/grid_client/scripts/algorand.ts:113]: added in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1409/commits/b899322cc957a8cb0501bb3c8ce49f0ed59969c5

**Pull request merged**